db/models/user.js

```diff
@@ -0,0 +1,53 @@
+var mongoose = require('mongoose');
+//add bcrypt and bluebird if not using Passport
+var bcrypt = require('bcrypt-nodejs');
+var Promise = require('bluebird');
+
+var userSchema = new mongoose.Schema({
+  name: String,
+  username: {
+    type: String,
+    required: true
+  },
+
+  //or are we using Passport?
+  password: {
+    type: String,
+    required: true
+  },
+
+  location: {
+    street: String,
+    city: String,
+    state: String,
+    //zipcode a string b/c some states have codes beginning with 0
+    zip: String
+  },
+  latitude: Number,
+  longitude: Number,
+});
```

Comment:
For now, I will store users' ' keywords' as an array. let's iron out the keyword associations tomorrow morning as a group to see what the backend team wants